osgp/platform/osgp-domain-core/src/main/java/org/opensmartgridplatform/domain/core/valueobjects/smartmetering/SmartMeteringDevice.java

```diff
@@ -36,6 +36,10 @@ public class SmartMeteringDevice implements Serializable {
   private byte[] globalEncryptionUnicastKey;
 
   private byte[] authenticationKey;
+  private byte[] globalEncryptionBroadcastKey;
```

Comment:
Where do you use globalEncryptionBroadcastKey?

Reply:
The unused key is removed.